idd/Energy+.idd.in

```diff
@@ -23825,14 +23825,15 @@ AirflowNetwork:MultiZone:WindPressureCoefficientValues,
       \note Enter the WPC Value corresponding to the 36th wind direction.
 
 AirflowNetwork:ZoneControl:PressureController,
-	 \memo Define the Pressures control settings for a zone or list of zones.
-	 \memo If you use a ZoneList in the Zone or ZoneList name field then this definition applies
-	 \memo to all the zones in the ZoneList.
+      \memo This object is used to control a zone to a specified indoor pressure 
+      \memo using the AirflowNetwork model. The specified pressure setpoint is used
+      \memo to control the zone exhaust fan flow rate in a controlled zone or 
+      \memo the relief air flow rate in an air loop.
       \unique-object
  A1 , \field Name
       \required-field
       \reference AirflowNetworkZoneControlPressureControllerNames
- A2 , \field Zone Name
+ A2 , \field Controlled Zone Name
```

Comment:
There are three places with "Controlled Zone Name" and everywhere else is "Control Zone Name".  So, yes, please change to "Control Zone Name" here.  If you want to clean up the others, the one in ZoneControl:ContaminantController should be "Zone Name" to match the pattern of thermostat objects.  And the one in AvailabilityManager:HybridVentilation should be "Control Zone Name".